cli/run.go

```diff
@@ -0,0 +1,70 @@
+package cli
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/spf13/cobra"
+	"k8s.io/kubectl/pkg/util/templates"
+	"os"
+)
+
+// RunOptions describe configuration options that can
+// be set using the run subcommand.
+type RunOptions struct {
+	*RootOptions
+	Config string
+}
+
+var clientRunExamples = templates.Examples(
+	`
+	# Push artifacts
```

Comment:
Since the name of the client executable changed since this PR was opened, can you change the examples to match the others? https://github.com/uor-framework/uor-client-go/blob/d16d41f1ce9bfbaa3c692b6b985ba9a46de85a2e/cli/build.go#L37. Other than that, this LGTM.